### PR TITLE
DISTMYSQL-357 Release notes for Percona Distribution for MySQL 8.0.34 (PXC)

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
new file:   docs/release-notes-pxc-v8.0.34.md
modified:   docs/release-notes.md
modified:   mkdocs-base.yml
modified:   docs/installing.md

### CODE REVIEW

**Review by patrickbirch — APPROVED** [review]
good

File: docs/release-notes-pxc-v8.0.33.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Percona Distribution for MySQL is the most stable, scalable and secure open-source MySQL distribution, with two download options: one based on Percona Server for MySQL and one based on Percona XtraDB Cluster. [Install Percona Distribution for MySQL](installing.md).
 
-This release is focused on the Percona XtraDB Cluster-based deployment variation. It is based on [Percona XtraDB Cluster 8.0.33](https://www.percona.com/doc/percona-xtradb-cluster/8.0/release-notes/8.0.33.html)
+This release is focused on the Percona XtraDB Cluster-based deployment variation. It is based on [Percona XtraDB Cluster 8.0.33](https://docs.percona.com/percona-xtradb-cluster/8.0/release-notes/8.0.33-25.html)
```
Comment (eleo007) [inline docs/release-notes-pxc-v8.0.33.md:5]: Isn't it 8.0.34?

File: docs/installing.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Install Percona Distribution for MySQL
 
+!!! note
```
Comment (patrickbirch) [inline docs/installing.md]: Doesn't need to be a note.

File: docs/installing.md [hunk 3]
```diff
@@ -1,5 +1,9 @@
...
+
+    We gather [Telemetry data on Percona XtraDB Cluster] in the Percona packages and Docker images.
+
 We recommend to install Percona Distribution for MySQL from Percona repositories using the package manager of your operating system:
```
Comment (patrickbirch) [inline docs/installing.md]: to install
installing the

File: docs/installing.md (hunk 2 above)
Comment (alina-derkach-oaza) [reply docs/installing.md]: fixed

File: docs/installing.md (hunk 3 above)
Comment (alina-derkach-oaza) [reply docs/installing.md]: fixed

**Pull request merged**